blackbox/docs/configuration.txt

```diff
@@ -1582,6 +1582,25 @@ Metadata Gateway
   number of nodes in the cluster.
 
 
+.. _conf-session-settings:
+
+Session Setting Parameters
+==========================
+
+The section lists the session setting parameters supported by Crate.
+Parameters can be set with the ``SET/SET SESSION`` statement, see :ref:`ref-set`.
+
+**search_path**
+  | *Default:* ``doc``
+  | *Runtime:* ``yes``
+
+  The parameter holds the schema search path. The value of ``search_path`` can
```

Comment:
This parameter holds the default schema for a session that should be used when no schema is provided in queries (or does it also override schema if provided in the statement?).
The value of `search_path` can be either a string or a list of strings. However, Crate only considers the first element when a list is provided.